### PR TITLE
bsunanda:Phase2-gem13 Correct forward muon part for 3 2023 scenario

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py [hunk 1]
```diff
@@ -233,7 +233,7 @@
         'Geometry/MuonCommonData/data/v7/gem21.xml',
         'Geometry/MuonCommonData/data/v2/csc.xml',
         'Geometry/MuonCommonData/data/PhaseII/mfshield.xml',
-        'Geometry/MuonCommonData/data/PhaseII/me0.xml',
+        'Geometry/MuonCommonData/data/v8/me0.xml',
```
Comment (ianna) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py]: @bsunanda - one may assume that if me0 is defined in v8, gem should also come from v8. Is there any reason to keep PhaseII/me0? Perhaps, replacing it with v8 would be better? Thanks.

Comment (calabria) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py]: @ianna - v8 does not exist for ge11, v7 is the final geometry for it. for ge21, v8 is only a temporary geometry we need for our studies, while v7 is the baseline, so it is better to go with v7 for the moment. when we have the final geometry for ge21 we will move to that. for me0, instead, v8 is an already improved version wrt the one present in the PhaseII folder. After this PR we will clean up a bit the old geometries, and organize the remaining files in folders with more explanatory names.

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@davidlange6 I tried to correct the files but I do not see the changes get updated. Is it because of the status?

**Comment by davidlange6** [comment]
this pr was merged already, so you need to make a new one instead of updating this one

**Pull request merged**